test/intersections.jl

```diff
@@ -1149,22 +1149,22 @@ end
   poly = Triangle(cart(6, 2), cart(3, 5), cart(0, 2))
   other = Quadrangle(cart(5, 0), cart(5, 4), cart(0, 4), cart(0, 0))
   @test intersection(poly, other) |> type == Intersecting
-  @test all(vertices(poly ∩ other) .≈ [cart(5, 3), cart(4, 4), cart(2, 4), cart(0, 2), cart(5, 2)])
+  @test all(vertices(poly ∩ other) .≈ [cart(2, 4), cart(0, 2), cart(5, 2), cart(5, 3), cart(4, 4)])
 
   # octagon
   poly = Octagon(cart(8, -2), cart(8, 5), cart(2, 5), cart(4, 3), cart(6, 3), cart(4, 1), cart(2, 1), cart(2, -2))
   other = Quadrangle(cart(5, 0), cart(5, 4), cart(0, 4), cart(0, 0))
   @test intersection(poly, other) |> type == Intersecting
-  @test all(
-    vertices(poly ∩ other) .≈
-    [cart(3, 4), cart(4, 3), cart(5, 3), cart(5, 2), cart(4, 1), cart(2, 1), cart(2, 0), cart(5, 0), cart(5, 4)]
-  )
+  out = vertices.(poly ∩ other)
+  @test length(out) == 2
+  @test all(out[1] .≈ [cart(3, 4), cart(4, 3), cart(5, 3), cart(5, 4)])
+  @test all(out[2] .≈ [cart(5, 2), cart(4, 1), cart(2, 1), cart(2, 0), cart(5, 0)])
 
   # inside
   poly = Quadrangle(cart(1, 0), cart(1, 1), cart(0, 1), cart(0, 0))
   other = Quadrangle(cart(5, 0), cart(5, 4), cart(0, 4), cart(0, 0))
   @test intersection(poly, other) |> type == Intersecting
-  @test all(vertices(poly ∩ other) .≈ vertices(poly))
+  @test all(vertices(poly ∩ other) .≈ [cart(1, 0), cart(1, 1), cart(0, 1), cart(0, 0)])
```

Comment:
```suggestion
  @test poly ∩ other) ≗ poly
```

Maybe we should adjust some of these tests to use `\circeq` between polygons.

Reply:
Done. Used it also in some other places where I could see it applicable. Probably not an exhaustive fix.